docs/configuration/git-providers/monitoring-conf/loki-values.yaml

```diff
@@ -0,0 +1,24 @@
+resources:
```

Comment:
To consider: we are providing all the template .yaml files under the directory /codacy. Would it make sense to include these there as well, as it makes it easier for user to find all templates in the same place? I'm especially worried that right now these files are buried quite deep inside the docs directory. :)

Reply:
What we are doing in the documentation for this is installing these components as a separate thing from codacy.

This line installs loki by itself and in a different namespace different than codacy:
`helm upgrade --install --atomic loki loki/loki --version 0.17.0 \
  --recreate-pods --namespace monitoring --values loki-values.yaml`

This "yaml" file is used to configure this new system.

Reply:
Yes, it is a good idea, but there's one caveat: the things inside `codacy/` get packaged into the chart. Not sure if it is wise to keep adding everything there. As the amount of extra stuff will keep growing that folder will get messier. For now it is probably fine, but in the future we should find a way to make it a bit more tidy. Maybe have a folder in the root of the repo where we collect all the extra stuff that most people won't need.